tests/locallib_test.php

```diff
@@ -257,6 +259,11 @@ public function test_guide() {
      */
     public function test_rubric_ranges() {
         $this->resetAfterTest();
+
+        if (core_component::get_plugin_directory('gradingform', 'rubric_ranges') === null) {
+            $this->markTestSkipped('Rubric ranges plugin not installed');
```

Comment:
Nice!. I will be adding test support for my BTEC plugin and I will create similar code for that.

Reply:
I'm not actually sure if this is the recommended way to check that (or if there even is one) but it works quite well.

Reply:
Looks good to me